proposals/0000-proposal-process.rst

```diff
@@ -0,0 +1,307 @@
+  benefit for these to be treated with a formal process; existing code review
+  processes are quite sufficient.
+
+* *Changes to the Haskell Report*: Language standardization has rather different
+  needs from the more exploratory proposals which GHC often entertains. Language
+  standards are formal specifications which only describes very well-understood
+  features; consequently the revision duration is significantly longer than the
+  typical GHC feature proposal. For this reason, it makes sense to keep
+  responsibility over standardization where it lies currently, with the Haskell
+  Prime Committee.
+
+* *Changes to the core libraries*: While there is often significant overlap
+  between language design and library design, we are going to declare this out
+  of scope for the time being.
+
+Motivation
+----------
+
+Let's start by examining a few of the shortcomings of GHC's current
+change management processes,
+
+1. Higher than necessary barrier-to-entry.
+
+   Involving oneself in the proposal process requires following a Trac ticket,
+   a Wiki page, one or more mailing list threads, and possibly Phabricator,
+   each of which requires a separate account
+
+2. The proposal itself and the discussion surrounding it are quite decoupled.
+
+   The proposal specification itself is kept on the Trac Wiki, design
+   discussion occurs in largely independent Trac tickets, and further
+   discussion is fragmented across several mailing lists, Reddit, and IRC. This
+   leads to repetitive and often difficult-to-follow discussions.
+
+3. The phases of the lifecycle of a proposal are not clearly defined.
+
+   This leads to unfortunate cases like that of ``ShortImports``, where
+   discussion proceeds for too long without a concrete specification. This
+   drains effort from both the author of the proposal and those discussing it.
+
+4. The process that exists isn't always consistently applied.
+
+   While this arguably has improved in recent years, there are cases where
+   features have been added without a formal specification or perhaps less
+   discussion than would be desired.
+
+
+Proposed change
+---------------
+
+I propose that we adopt a variant on the
+`Rust RFC scheme <https://github.com/rust-lang/rfcs#what-the-process-is>`_,
+building our proposal process on top of existing code review tools for
+dissemination and refinement of proposals with a standing committee acting as
+the final arbiter of proposal acceptance.
+
+To ensure that the process is as accessible and convenient as possible, I
+propose that we build this process on top of Github.
+
+Proposal process
+~~~~~~~~~~~~~~~~
+
+Each proposal goes through the following stages:
+
+1. The process begins with the opening of a pull request to the GHC proposals
+   repository, proposing the merge of a document (derived from a
+   `provided template <https://github.com/bgamari/ghc-proposals/blob/master/0000-template.rst>`_)
```

Comment:
That URL should be https://github.com/ghc-proposals/ghc-proposals/blob/master/0000-template.rst